src/hacking/building-for-openharmony.md

```diff
@@ -4,189 +4,29 @@
-        "keyAlias": "debugKey",
-        "keyPassword": "<encrypted password>",
-        "profile": "/path/to/signed-profile-certificate.p7b",
-        "signAlg": "SHA256withECDSA",
-        "storeFile": "/path/to/java-keystore-file.p12"
-    }
-}
-```
-
-Here `<encrypted password>` is a hexadecimal string representation of the plaintext password after being encrypted.
-The key and salt used to encrypt the passwords are generated by DevEco Studio IDE and are stored on-disk alongside the certificate files and keystore, usually under `<USER HOME>/.ohos/config/openharmony`.
-
-You can use the IDE to generate the information needed for password encryption, the required application and profile certificate files, and the keystore itself.
-
-1. Open Project Structure dialog from `File > Project Structure` menu.
-2. Under the 'Signing Config' tab, enable the 'Automatically generate signature' checkbox.
-
-**NOTE: The signature autogenerated above is intended only for development and testing. For production builds and distribution via an App Store, the relevant configuration needs to be obtained from the App Store provider.**
-
-Once generated, it is necessary to point `mach` to the above "signing material" configuration using the `SERVO_OHOS_SIGNING_CONFIG` environment variable.
-The value of the variable must be a file path to a valid `.json` file with the same structure as the `signingConfigs` property given above, but with `certPath`, `storeFile` and `profile` given as *paths relative to the json file*, instead of absolute paths.
-
-
-## Building servoshell
+### Building servoshell
 
 Before building servo you will need to set some environment variables.
 [direnv](https://direnv.net) is a convenient tool that can automatically set these variables based on an `.envrc` file, but you can also use any other method to set the required environment variables.
 
 `.envrc`:
 ```commandline
     export OHOS_SDK_NATIVE=/path/to/openharmony-sdk/platform/api-version/native
-
-    # Required if the HAP must be signed. See the signing configuration section above.
-    export SERVO_OHOS_SIGNING_CONFIG=/path/to/signing-configs.json
-
-    # Required only when building for HarmonyOS:
-    export DEVECO_SDK_HOME=/path/to/command-line-tools/sdk # OR /path/to/DevEcoStudio/sdk OR on MacOS /Applications/DevEco-Studio.app/Contents/sdk
-
-    # Required only when building for OpenHarmony:
-    # Note: The openharmony sdk is under ${DEVECO_SDK_HOME}/default/openharmony
-    # Presumably you would need to replicate this directory structure
     export OHOS_BASE_SDK_HOME=/path/to/openharmony-sdk/platform
-
-    # If you have the command-line tools installed:
-    export PATH="${PATH}:/path/to/command-line-tools/bin/"
-    export NODE_HOME=/path/to/command-line-tools/tool/node
-
-    # Alternatively, if you do NOT have the command-line tools installed:
     export HVIGOR_PATH=/path/to/parent/directory/containing/node_modules  # Not required if `hvigorw` is in $PATH
```

Comment:
Please revert the envrc changes. The exact variables needed depend on your configuration, and e.g. removing DEVECO_SDK_HOME will break packaging for HarmonyOS.